### PR TITLE
Cloud docs

### PR DESCRIPTION
**Pull request by sguerciojr**
added some screen shots and documentation on cloud basic operations.
added subscription first pass options.

### CODE REVIEW

File: pages/cloud/using.md [hunk 1]
```diff
@@ -1,5 +1,284 @@
...
+
+[The Lake Effect Dashboard (Horizon)](https://dashboard.cloud.ccr.buffalo.edu) - simplifies management of your project's cloud resources and provides information on your group's usage.
+
+![](../images/cloud/openid.png)
+
+
+You will need to authenticate to CCR's **Identity Management System** with your CCR username and password+otp
+For more information about logging in with two-factor authentication enabled click [here](../2fa.md).
+
+![](../images/cloud/ccridm.png)
```
Comment (dsajdak) [inline pages/cloud/using.md:17]: I would recommend removing this screenshot as the login process will be changing shortly and looks different.  Users will be prompted to enter their username, then taken to the next page where they enter their password and, in a separate box, OTP code.

File: pages/cloud/lake-effect.md [hunk 2]
```diff
@@ -10,38 +10,62 @@ Although high performance computing (HPC) is our largest and most widely used se
 
 ### Lakeeffect Quick Start
```
Comment (dsajdak) [inline pages/cloud/lake-effect.md:11]: A nit-picky comment but suggest using the same formatting throughout the docs for our naming of things.  Mostly you have LakeEffect, here it's Lakeeffect.  Some places we have coldfront or Coldfront but we've been trying to use ColdFront everywhere

Comment (dsajdak) [inline pages/cloud/lake-effect.md:11]: A nit-picky comment but suggest using the same formatting throughout the docs for our naming of things.  Mostly you have LakeEffect, here it's Lakeeffect.  Some places we have coldfront or Coldfront but we've been trying to use ColdFront everywhere

File: pages/cloud/lake-effect.md [hunk 3]
```diff
@@ -1,47 +1,71 @@
...
 
-Although high performance computing (HPC) is our largest and most widely used service, CCR recognizes that not all research requires number crunching on large scale computers (clusters). Lake Effect, was put into service to provide flexible infrastructure to support research computing such as webservers, databases, and long-running services that are not suited for HPC clusters. This along with our HPC platform allows us to provide a complete set of tools to support all the different research groups and collaborations througout UB.
+Although high performance computing (HPC) is our largest and most widely used service, CCR recognizes that not all research requires number crunching on large scale computers (clusters). LakeEffect, was put into service to provide flexible infrastructure to support research computing such as webservers, databases, and long-running services that are not suited for HPC clusters. This along with our HPC platform allows us to provide a complete set of tools to support all the different research groups and collaborations througout UB.
```
Comment (dsajdak) [inline pages/cloud/lake-effect.md:8]: throughout is misspelled in the last sentence.  Sorry I missed that before

File: pages/cloud/lake-effect.md [hunk 4]
```diff
@@ -1,47 +1,71 @@
...
 
-1. Details on Pricing and Subscription packages is [here](subscriptions.md)
+1. Details on Pricing and Subscription packages is [here](#subscriptions)
 2. Request access through the [coldfront](../portals/coldfront.md) allocations management portal
 3. Once you have an active allocation, you can access the cloud through the [Dashboard](https://dashboard.cloud.ccr.buffalo.edu)
-4. Documenation on using Lakeeffect Cloud and Openstack:
-	- [Using LakeEffect Cloud](./using.md)
-	- [OpenStack Documentation](https://docs.openstack.org/)
-
+4. Documenation on using LakeEffect Cloud and Openstack:
```
Comment (dsajdak) [inline pages/cloud/lake-effect.md:16]: Documentation is misspelled

File: pages/cloud/lake-effect.md [hunk 5]
```diff
@@ -1,47 +1,71 @@
...
-- Load balancing of virtual resources across multiple physical machines
-- Support for Floating IP addresses
-- Easily manage access to VMs using Security Groups and Rules
-- Built on Openstack Cloud Software [Openstack](https://www.openstack.org/) 
-- Fault tolerant components to minimize risk of downtime due to underlying hardware failure
+* Deploy virtual servers in minutes
+* Storage volume creation/snapshot
+* VPC Networking with access to public IP address space as well as the ability to create private networks
+* Wide range of Machine Types available inclunding GPUs and IO Optimized Storage
+* Various Linux operating systems are supported.  _**NOTE**: Windows images are not provided be CCR at this time_
```
Comment (dsajdak) [inline pages/cloud/lake-effect.md:28]: should be 'are not provided by'

File: pages/cloud/lake-effect.md [hunk 6]
```diff
@@ -1,47 +1,71 @@
...
+* Public Images of opensource Linux Distributions available as well as the ability for users to upload their own
+* Load balancing of virtual resources across multiple physical machines
+* Support for Floating IP addresses
+* Easily manage access to VMs using Security Groups and Rules
+* Built on Openstack Cloud Software [Openstack](https://www.openstack.org/)
```
Comment (dsajdak) [inline pages/cloud/lake-effect.md:33]: This link is kind of repetitive.  Did you mean to make "Openstack Cloud Software" the link?

File: pages/cloud/using.md [hunk 7]
```diff
@@ -1,5 +1,289 @@
...
+!!! Note
+    Before you can launch and login to instances, you must have setup a key pair as described [here](#keypairs)
+
+
+From the LakeEffect Dashboard go to **Compute -> Instances**
+
+Click on `Launch Instance` button at the top right corner. This will bring up the instance launcher which will guide you in creating your new instance by selecting your instance source, Flavor, Network, Security groups, and Keypair.
+
+!!! Note
+    _Your project has a quota set for the total number of instances you're permitted to run.  The chart on the right shows the current usage.  If you've hit your limit, please contact ccr-help@buffalo.edu to request assistance._
```
Comment (dsajdak) [inline pages/cloud/using.md:192]: we're adding a top level page called help.md so anywhere we say 'contact CCR Help' we can link to that

File: pages/cloud/using.md [hunk 8]
```diff
@@ -1,5 +1,289 @@
...
+![](../images/cloud/instance-security.png)
+
+!!! Tip
+    Security groups act at the instance level and can be changed after instance creation.
+
+> * click `Next` when done to go to the ___Key Pair___ screen.
+
+##### Step 6: Key Pair
+
+**Key Pair** allows you to SSH into your instance.  You may select an existing key pair, import a key pair you've already created (see the previous section on [Key Pairs](#keypairs), or generate a new key pair from this tab.
```
Comment (dsajdak) [inline pages/cloud/using.md:269]: missing the closing ) after keypairs

File: pages/cloud/lake-effect.md [hunk 9]
```diff
@@ -1,47 +1,71 @@
...
+| Standard| 20 vCPU | 2 TB | 2 Floating IP | $1117.62/year |
+| Advanced| 40 vCPU | 4 TB | 4 Floating IP , Fast-IO VMs | $2235.24/year |
+| GPU Addon| 8 vCPU | 1 TB | 1 Floating IP, Tesla V100 16GB | $936.34/year |
+| Storage addon | | 1 TB | Block Storage | $54.36/TB/year |
+
+* Systems Administration Consulting: $59.97/hour
+
+    * Must be purchased in 8 hour increments
+    * Can be used in 30 minute chunks
+    * Contact ccr-help for estimates on required time
```
Comment (dsajdak) [inline pages/cloud/lake-effect.md:65]: 'Contact CCR Help' link to new help.md file to be created - here and in the tip below

File: pages/cloud/using.md (hunk 1 above)
Comment (sguerciojr) [reply pages/cloud/using.md]: stuck in a new screenshot.

File: pages/cloud/lake-effect.md (hunk 2 above)
Comment (sguerciojr) [reply pages/cloud/lake-effect.md]: all fixed up and made consistant

File: pages/cloud/lake-effect.md (hunk 3 above)
Comment (sguerciojr) [reply pages/cloud/lake-effect.md]: fixed.

File: pages/cloud/lake-effect.md (hunk 4 above)
Comment (sguerciojr) [reply pages/cloud/lake-effect.md]: fixed

File: pages/cloud/lake-effect.md (hunk 6 above)
Comment (sguerciojr) [reply pages/cloud/lake-effect.md]: fixed - made "Openstack Cloud Software" the link

File: pages/cloud/using.md (hunk 7 above)
Comment (sguerciojr) [reply pages/cloud/using.md]: replaced all the ccr-help@buffalo.edu with a link to the help doc.

File: pages/cloud/using.md (hunk 8 above)
Comment (sguerciojr) [reply pages/cloud/using.md]: fixed

File: pages/cloud/lake-effect.md (hunk 9 above)
Comment (sguerciojr) [reply pages/cloud/lake-effect.md]: replaced all the ccr-help@buffalo.edu with a link to the help doc.

**Review by dsajdak — APPROVED** [review]
LGTM!

**Pull request merged**